<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="up">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="up2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Up2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="down2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Down2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 1 Buttons</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 2 Buttons</string>
    </property>
   </widget>
   <widget class="QPushButton" name="up3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Up3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="down3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Down3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="down4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Down4</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>332</width>
      <height>227</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Elevator1">
     <attribute name="title">
      <string>Elevator 1</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="0">
       <widget class="QPushButton" name="floor1">
        <property name="text">
         <string>Floor 1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="floor4">
        <property name="text">
         <string>Floor 4</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="Overload">
        <property name="text">
         <string>Overload</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="floor2">
        <property name="text">
         <string>Floor 2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="closeDoor">
        <property name="text">
         <string>CloseDoor</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="help">
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="DoorObs">
        <property name="text">
         <string>DoorObstacle</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="floor3">
        <property name="text">
         <string>Floor 3</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="Fire">
        <property name="text">
         <string>Fire</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="openDoor">
        <property name="text">
         <string>OpenDoor</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Location &amp; Direction</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="e1currentPos">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Current Positon</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="e1currentDir">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Current Direction</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="2" rowspan="2">
       <widget class="QLabel" name="EmergMessage">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Emerg Message</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Emergency Display</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="Elevator2">
     <attribute name="title">
      <string>Elevator 2</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="e2floor4">
        <property name="text">
         <string>Floor 4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="e2OpenDoor">
        <property name="text">
         <string>OpenDoor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Location &amp; Direction</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="e2floor3">
        <property name="text">
         <string>Floor 3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="e2CloseDoor">
        <property name="text">
         <string>CloseDoor</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="e2currentPos">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Current Positon</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="e2floor2">
        <property name="text">
         <string>Floor 2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="e2Help">
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="e2currentDir">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Current Direction</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="e2floor1">
        <property name="text">
         <string>Floor 1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="e2Fire">
        <property name="text">
         <string>Fire</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Emergency Display</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="e2doorObs">
        <property name="text">
         <string>DoorObstacle</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" rowspan="2">
       <widget class="QLabel" name="e2EmergMessage">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Emerg Message</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="e2Overload">
        <property name="text">
         <string>Overload</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 3 Buttons</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 4 Buttons</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Strategy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>565</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
